src/vm_pages.rs

```diff
@@ -730,17 +775,53 @@ impl<T: GuestStagePagingMode, S> VmPages<T, S> {
         Ok(())
     }
 
-    /// Locks `count` 4kB pages starting at `page_addr` for mapping in a region of type
-    /// `region_type`, returning a `VmPagesMapper` that can be used to insert (and measure, if
-    /// necessary) the pages.
-    pub fn map_pages(
+    // Locks `count` 4kB pages starting at `page_addr` for mapping in a region of type
+    // `region_type`, returning a `VmPagesMapper` that can be used to insert (and measure, if
+    // necessary) the pages.
```

Comment:
nit: the descriptions of the args is lost from the public docs, maybe move to map_zero_pages and reference it from the others?

Reply:
Done.